scripts/generateDocs.js

```diff
@@ -1,30 +1,32 @@
 const path = require('path');
-const chalk = require('chalk');
 const markdownMagic = require('markdown-magic');
 const fs = require('fs');
 
 const config = {
   matchWord: 'AURO-GENERATED-CONTENT'
-}
+};
```

Comment:
This whole document is incorrect? Please use code from https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/generateDocs.js

Also, when code is updated note the error of missing resources that were deleted in a previous commit.

Reply:
Hi @blackfalcon,
I will fix your feedback, thank you.